src/pages/index.js

```diff
@@ -77,15 +134,28 @@ profileAddButton.addEventListener("click", () => {
 const profileEditPopup = new PopupWithForm(
   {
     popupSelector: "#profile-edit-modal",
-    handleFormSubmit: ({ title, description }) => {
-      profileInfo.setUserInfo(title, description);
-      profileEditPopup.close();
-    }
+    handleFormSubmit: ({ title, description }, profileEditForm) => {
```

Comment:
`profileEditForm` 

It's a submit button, actually, because you pass it from `PopupWithForm`. But you don't need it

You shouldn't reset the profile form after submission - it has no effect


```js
    handleFormSubmit: ({ title, description }) => {


```